test/Table_Tests/src/IO/Delimited_Read_Spec.enso

```diff
@@ -398,6 +398,15 @@ spec =
             t2.at "c" . to_vector . should_equal [3, 6]
             t2.columns.map .name . should_equal ["a", "b", "c"]
 
+        Test.specify "should be able to read in a file without splitting it to columns" <|
+            t1 = (enso_project.data / "data_small.csv") . read (Delimited "" headers=False)
+            expected = ['Code,Index,Flag,Value,ValueWithNothing,TextWithNothing,"Hexadecimal",Leading0s,QuotedNumbers,"Mixed Types"']
+            expected2 = expected + ['gxl,7,True,38.76109,63.13,   pq6igd2wyd  ,4DD4675B,001,"1","33"']
+            expected3 = expected2 + ['wca,0,False,-66.77495,31,"  2pr4102wc4  ",,002,"2",']
+            expected4 = expected3 + ['nfw,1,  True ,  88.65713\t\t\t,-68.71,"",01896EAB,123,,45']
+            expected5 = expected4 + ['der,7,True,0.86658,,,F32E1EFE,,"34",True']
```

Comment:
Any cleaner way of building this large vector up? I'm not very happy with this syntax, but I think it is better than one long line...

Reply:
Indeed multiline vector syntax does not seem to work yet.

I think what I'd do is following:
```
vb = Vector.new_builder
vb.append "line1"
vb.append "line2"
...
vb.to_vector ...
```

It is not perfect, it is awfully mutable and impure. But it is readable and works :)

Reply:
I just tried one more trick. Given how vector is implemented it will be O(N^2) same as the first solution (but for this small test that is probably not at all a problem).

It relies on #8415 (thanks @JaroslavTulach 🎉):

```
from Standard.Base import all

main =
    v = [1]
        + [2]
        + [3]
        + [4]
    IO.println v
```
actually works and prints
```
[1, 2, 3, 4]
```

I think it may be a viable alternative to the vector builder approach. IMHO it looks a little bit cleaner than the builder, but I'm not 100% convinced which one is better. I guess both seem okay.